src/pages/Homepage/Homepage.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+	<www-page>
+		<div class="row page-content">
+			<div class="small-12 columns">
+				<h1>NEW UI HOMEPAGE</h1>
+			</div>
+		</div>
+	</www-page>
+</template>
+
+<script>
+import WwwPage from '@/components/WwwFrame/WwwPage';
+
+export default {
+	components: {
+		WwwPage
+	},
+	metaInfo: {
+		title: 'Sitemap'
```

Comment:
For the homepage I don't think we need to specify any title and we can just let the default be used

Reply:
My bad, I just remove this copy/paste error.